lib/fileutils.rb

```diff
@@ -1362,7 +1372,7 @@ def fu_stat_identical_entry?(a, b)   #:nodoc:
   # Optional argument +force+ specifies whether to ignore
   # raised exceptions of StandardError and its descendants.
   #
-  # Related: FileUtils.remove_entry_secure.
```

Comment:
I think we should keep this, since `FileUtils.remove_entry_secure` is so closely related to `FileUtils.remove_entry`.